modules/op-resolver-stepactions-ref.adoc

```diff
@@ -0,0 +1,79 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: REFERENCE
+[id="resolver-stepactions-ref_{context}"]
+= Step action provided with {pipelines-shortname}
+
+{pipelines-shortname} provides a standard `StepAction` definition that you can use in your tasks. Use the HTTP resolver to reference this definition.
+
+[discrete]
+[id="op-stepaction-git-clone_{context}"]
+== git-clone
+
+The `git-clone` step action uses Git to initialize and clone a remote repository on a workspace. You can use this step action to define a task that clones a repository at the start of a pipeline that builds or otherwise processes this source code.
+
+To reference this step action in a task, use the HTTP resolver with the following URL: `https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml`.
+
+.Example usage of the `git-clone` step action in a task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: clone-repo-anon
+spec:
+  params:
+  - name: url
+    description: The URL of the Git repository to clone
+  workspaces:
+  - name: output
+    description: The git repo will be cloned onto the volume backing this Workspace.
+  steps:
+  - name: clone-repo-anon-step
+    ref:
+      resolver: http
+      params:
+      - name: url
+        value: https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml
+    params:
+    - name: URL
+      value: $(params.url)
+    - name: OUTPUT_PATH
+      value: $(workspaces.output.path)
+----
+
+.Supported parameters for the `git-clone` step action
```

Comment:
```suggestion
.Supported parameters for the `git-clone` StepAction
```

Reply:
The style guide does not permit us to use `StepAction` as a standalone noun. It can be a StepAction definition, StepAction resource, and so on, but when it's a standalone thing, it's "step action".